docs/source/operations/fabric_verify.md

```diff
@@ -0,0 +1,167 @@
+If not, Check flux-helm operator logs..
+Possible errors
+- Value files are not generated right. Release failed
+For further details check the pod logs or describe the helm release files
+
+Stage 6.
+Create channel/ Join channel
+Check if pod is up and running, If not Check Flux logs
+Possible errors:
+- Error: failed to create deliver client: orderer client failed to connect to <url>:7050: failed to create new connection: context deadline exceeded
+
+Not able to reach orderer, Check orderer logs for more details.
+Backtrace it, check if ambassador is working 
+
+- got unexpected status: BAD_REQUEST -- error authorizing update: error validating DeltaSet: policy for [Group] /Channel/Application not satisfied: Failed to reach implicit threshold of 1 sub-policies, required 1 remaining
+
+The most common reasons for are:
+
+a) This identity is not in the organization's administrator list.
+
+b) The organization certificate is not effectively signed by the organization's CA chain.
+
+c) The organization that the orderer does not know about identity.
+
+Stage 8. 
+Achor Peers
+
+Check peer logs, filter results using grep "anchor"
+If anchor peers are successful, you should find the list in the peer logs.
+
+     kubectl logs <pod name> -n <namespace> | grep anchor
+The output should have the list of the anchor peers.
+ 
+Stage 9.
+Install Chaincode/ Instantiate Chaincode
+Check Flux logs.
+If Pod is not up and running backtrace to which component is creating issue.
+
+
+**NOTE**:
+If the components are not able to connect to each other, there could be some issue  with load balancer. Check the haproxy or external DNS logs for more debugging.
+
+##### Final network validity check.
+
+For final checking of the validity of the fabric network.
+
+1. Create a cli pod for any organization.Use this sample template. 
+
+
+    metadata:
+      namespace: <namespace>
+      images:
+        fabrictools: hyperledger/fabric-tools:2.0
+        alpineutils: index.docker.io/hyperledgerlabs/alpine-utils:1.0
+    storage:
+      class: <storage class name>
+      size: 256Mi
+    vault:
+      role: vault-role
+      address: <vault address>
+      authpath: <auth path>
+      adminsecretprefix: <Path to admin secret Prefix>
+      orderersecretprefix: <Path to orderer secret prefix>
+      serviceaccountname: <Service account name>
+      imagesecretname: regcred
+      tls: false
+    peer:
+      name: <peer name>
+      localmspid: <msp id of peer>
+      tlsstatus: true
+      address: <peer address>
+    orderer:
+      address: <orderer address>
+
+2. To install the cli.
+
+
+    helm install -f cli.yaml <chart path> -n <namespace>
+
+3. Get the cli pod.
+
+    
+    kubectl get pod -n <namespace>
+
+4. Copy the cli pod name from the output list and enter the cli using.
+
+    
+    kubectl exec -it <cli pod name> -n <namespace> -- bash
+    
+5. To see which chaincodes are installed 
+
+
+    peer chaincode list --installed
+
```

Comment:
Also add the logic to check if the chaincode is instantiated or not
peer chaincode list --instantiated -C allchannel